app/templates/base.html

```diff
@@ -63,6 +63,26 @@
                     }
                 )
             });
+        function set_message_count(n) {
+            $('#message_count').text(n);
+            $('#message_count').css('visibility', n ? 'visible' : 'hidden');
+        }
+        {% if current_user.is_authenticated %}
+        $(function() {
+            var since = 0;
+            setInterval(function() {
+                $.ajax('{{ url_for('main.notifications') }}?since=' + since).done(
+                    function(notifications) {
+                        for (var i = 0; i < notifications.length; i++) {
+                            if (notifications[i].name == 'unread_message_count')
```

Comment:
it's better to use `===` instead of `==` to check equality
`==` checks value while `===` checks value and type
https://www.scaler.com/topics/javascript/difference-between-double-equals-and-triple-equals-in-javascript/#:~:text=Comparison%20and%20Difference%20Between%20%3D%3D%20and%20%3D%3D%3D%20in%20Javascript